lib/cmd/fh3/app/stage.js

```diff
@@ -19,12 +19,23 @@ base.describe['gitRef.value'] = i18n._("The name of the branch you'd like to dep
 base.alias.runtime = "r";
 base.alias.clean = "c";
 base.alias.autodeploy = "d";
+
+//Add custom example
+base.examples[0].cmd = 'fhc app stage --app=<appGuid> --env=<environmentName> (Defaults to branch:master)';
```

Comment:
Not sure if there is a better way to do this. I needed the usage to show that it defaults to master.